packages/openneuro-app/src/scripts/refactor_2021/search/search-container.tsx

```diff
@@ -92,11 +92,22 @@ const SearchContainer: FC<SearchContainerProps> = ({ portalContent }) => {
 
   const { loading, data, fetchMore, refetch, variables, error } =
     useSearchResults()
+  const loadMore = loading
+    ? () => {}
+    : () =>
+        fetchMore({
+          variables: {
+            // ...variables,
+            cursor: data?.datasets?.pageInfo.endCursor,
+          },
+        }).then((...args) => console.log(args))
```

Comment:
Looks like an extra console.log and both of these branches should return the same type (loadMore: void seems correct).